molecule/config.py

```diff
@@ -47,10 +47,12 @@ def populate_instance_names(self, platform):
         :param platform: platform name to pass to ``format_instance_name`` call
         :return: None
         """
-        for instance in self.config['vagrant']['instances']:
-            instance['vm_name'] = utilities.format_instance_name(
-                instance['name'], platform,
-                self.config['vagrant']['instances'])
+
+        if 'vagrant' in self.config:
```

Comment:
Yes. I was also running into an issue where molecule overwrites the ansible.cfg file after each run. I can no longer specify custom configs for changing role path. Going to try and fix it.